src/types.rs

```diff
@@ -28,7 +28,7 @@ impl fmt::Display for PlatformType {
 impl str::FromStr for PlatformType {
     type Err = String;
 
-    fn from_str(s: &str) -> Result<Self, Self::Err> {
+    fn from_str(s: &str) -> StdResult<Self, Self::Err> {
```

Comment:
Any particular reason this uses this instead of having `type Err = anyhow::Error`?

Reply:
Not really, we could change that.